mailet/api/pom.xml

```diff
@@ -55,11 +55,16 @@
             <version>1.7.5</version>
```

Comment:
Are you sure only:
`setMessage` and `getMessage().addHeader()` are the only think that can invalidate parsing ?